src/mame/drivers/mpu4avan.cpp

```diff
@@ -5,13 +5,30 @@
 
   split out because there might be hardware differences, and there are an awful lot of them, especially american beauty! ;-)
   I think some might be dual unit setups, some roms are marked top/bottom
+
+  are some of these actually by Concept?
 */
 
 #include "emu.h"
 #include "includes/mpu4.h"
 
 INPUT_PORTS_EXTERN( mpu4 );
 
+namespace {
+
+class mpu4avan_machines_state : public mpu4_state
+{
+public:
+
+	mpu4avan_machines_state(const machine_config& mconfig, device_type type, const char* tag) :
+		mpu4_state(mconfig, type, tag)
+	{
+	}
+};
+
+}; // anonymous namespace
```

Comment:
Closing braces of namespaces don’t need a semicolon (like functions, they aren’t “statements” from the compiler’s point of view, so they don’t need a terminating semicolon).  You may as well preempt @0kmg doing one of his “remove superfluous semicolons” on these files ;)